spec/lib/workable/client_spec.rb

```diff
@@ -18,15 +18,15 @@
 
   describe '#about' do
     it 'returns information about company as a hash' do
-      stub_request(:get, 'https://www.workable.com/spi/v3/accounts/subdomain/')
+      stub_request(:get, 'https://subdomain.workable.com/spi/v3/accounts/subdomain/')
```

Comment:
According to the docs at https://workable.readme.io/docs/accountssubdomain there should be no subdomain on this version of the URL. It's unclear whether it actually matters.